lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -894,13 +894,13 @@ def detect_init_system(self) -> InitMode:
             logging.info("systemd is not installed")
 
         if docker_running and systemd_running:
-            raise RuntimeError("Magmad is running with both Docker and systemd")
-        elif docker_running:
+            return InitMode.SYSTEMD  # default to systemd if both are running - needed by feg integ tests
+        if docker_running:
```

Comment:
Nitpick: The mix if-if-elif-else looks weird. I'd either do if-if-if-raise or if-elif-elif-else.